### PR TITLE
Credentials moved to the request body

### PR DESCRIPTION
**Pull request by alexey-rise**
## Description
Moved credentials to request body for all UserAuth API calls.

## Motivation and Context
As per vulnerability report the application exposes the user’s username and password used for authentication when logging into the application and when a user resets their password. Although both requests are using POST, the credentials are passed through the URL query rather than through the body. This allows the credentials to be stored in a multitude of unintended locations such as server logs, third-party sites (through the referrer header), proxies, and in the browser history. Attackers may also be able to obtain the credentials by sniffing the traffic of the victim through a Man-in-the-Middle (MitM) attack. This change eliminates this vulnerability.

## How Has This Been Tested?
Tested locally with Apps.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
N/A

### CODE REVIEW

**Comment by alexey-rise** [comment]
@tejohnso @alex-deaconu I suspect that some of the /dist/* changes are unnecessary (for example CSS or dependencies.js) and should be removed from the commit, please let me know if you agree.

**Comment by alex-deaconu** [comment]
@alexey-rise that is correct. None of those dist changes should be there other than:
`dist/js/components/userstate.js`
`dist/js/components/userstate.min.js`
`dist/js/common-header.js`
`dist/js/common-header.min.js`

Thanks.

**Comment by alex-deaconu** [comment]
Also, don't forget to bump the `bower.json`/`package.json` versions before merging. Thanks.

**Comment by alexey-rise** [comment]
@alex-deaconu I re-did the commit with only the necessary files.

**Comment by alex-deaconu** [comment]
@alexey-rise looks good, just curious what your plan for deployment is, considering this is a breaking change reliant on the updated Core version. Thanks.

**Comment by alexey-rise** [comment]
@alex-deaconu please see https://github.com/Rise-Vision/core/pull/538/commits/5f784f8eba307261f254c19612a88ebb39d9cbfb
The change is not so breaking anymore.

**Comment by alex-deaconu** [comment]
@alexey-rise that's great, thanks for addressing!

File: dist/js/common-header.js [hunk 1]
```diff
@@ -10163,20 +10178,20 @@ angular.module("risevision.common.components.purchase-flow")
             if (!paymentMethods.selectedCard.isNew) {
               deferred.resolve();
             } else {
-              var address = paymentMethods.newCreditCard.address;
-              if (paymentMethods.newCreditCard.useBillingAddress) {
+              var address = paymentMethods.newCreditCard && paymentMethods.newCreditCard.address;
```
Comment (alexey-rise) [inline dist/js/common-header.js]: @tejohnso did you make these changes? I didn't and they appeared after I rebased on top of your 3DS version.

Comment (tejohnso) [reply dist/js/common-header.js]: I did.

Comment (alexey-rise) [reply dist/js/common-header.js]: @tejohnso Hmm. Then how can they be in my commit that is supposed to be on top of yours which already has these changes? Or is this diff undoing your changes?

**Comment by alex-deaconu** [comment]
@tejohnso the diff issue is that your `dist` folder from https://github.com/Rise-Vision/common-header/pull/1055 does not contain all the changes in the PR. I'm wondering if this is causing any issues in production... @alexey-rise FYI. Thanks.

**Comment by tejohnso** [comment]
@alex-deaconu we discussed this morning. I forgot to merge some dist/ fixes related to unit tests in my last commit, so Alexey's build picked those changes up in dist/ and commited here. No issues.

**Pull request merged**